### ISSUE TITLE
_C问题

### ISSUE THREAD

**Issue by yzhyrz** [ViTAE-Transformer/DeepSolo#40]
Can it run on windows?

**Comment by sraimund** [comment]
Yes, try to install the Visual Studio Build Tools 2019 to build detectron2 and this project. This stackoverflow answer may also help: https://stackoverflow.com/a/77061123/1424765
